index.js

```diff
@@ -96,7 +103,7 @@ function MochaXUnitReporter(runner, options) {
   this._runner.on(
     'suite',
     function(suite) {
-      if (suite.tests.length) {
+      if (!isInvalidSuite(suite)) {
```

Comment:
Would it be less confusing to change the function to be `isValidSuite()`.
Double negative are never a good idea IMHO...

Reply:
```suggestion
      if (isValidSuite(suite)) {
```